src/SDKs/DataFactory/Management.DataFactory/changelog.md

```diff
@@ -12,6 +12,8 @@
   * Add Salesforce Sink
   * Add jsonNodeReference and jsonPathDefinition to JSONFormat
   * Support providing Salesforce passwords and security tokens as SecureStrings or as secrets in Azure Key Vault.
+  * Add SAP Cloud For Customer Source
+  * Add SAP Cloud For Customer Dataset
```

Comment:
version `0.2.1-preview` has already been published. Please create a new section for future version `0.2.2-preview`

Reply:
updated to be a new version